github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml

```diff
@@ -0,0 +1,126 @@
+presubmits:
+  kubevirt/managed-tenant-quota:
+  - name: pull-mtq-unit-test-1.1
+    context: pull-mtq-unit-test
+    branches:
+      - release-v1.1
+    cluster: ibm-prow-jobs
+    always_run: true
+    optional: false
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+    spec:
+      containers:
+        - image: quay.io/kubevirtci/bootstrap:v20220728-1410a63
+          command:
+            - "/usr/local/bin/runner.sh"
+            - "/bin/sh"
+            - "-c"
+            - "make test"
+          # docker-in-docker needs privileged mode
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              memory: "4Gi"
+  - name: pull-mtq-generate-verify-1.1
+    context: pull-mtq-generate-verify
+    branches:
+      - release-v1.1
+    cluster: ibm-prow-jobs
+    always_run: true
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    labels:
+      preset-dind-enabled: "true"
```

Comment:
Same here - is there a reason using for older dind preset?

Reply:
Done